src/commands/lambda/flare.ts

```diff
@@ -197,23 +256,39 @@ export class LambdaFlareCommand extends Command {
       }
 
       // Create folders
-      this.context.stdout.write('\n💾 Saving files...\n')
+      this.context.stdout.write(chalk.bold('\n💾 Saving files...\n'))
       const rootFolderPath = path.join(process.cwd(), FLARE_OUTPUT_DIRECTORY)
       const logsFolderPath = path.join(rootFolderPath, LOGS_DIRECTORY)
+      const projectFilesFolderPath = path.join(rootFolderPath, PROJECT_FILES_DIRECTORY)
+      const additionalFilesFolderPath = path.join(rootFolderPath, ADDITIONAL_FILES_DIRECTORY)
       if (fs.existsSync(rootFolderPath)) {
         deleteFolder(rootFolderPath)
       }
-      createDirectories(rootFolderPath, logsFolderPath, logs)
+      const subFolders = []
+      if (logs.size > 0) {
+        subFolders.push(logsFolderPath)
+      }
+      if (projectFilesToPath.size > 0) {
+        subFolders.push(projectFilesFolderPath)
+      }
+      if (additionalFiles.size > 0) {
+        subFolders.push(additionalFilesFolderPath)
+      }
+      createDirectories(rootFolderPath, subFolders)
```

Comment:
Is there a more concise way to do this? I couldn't really think of any other solution

Reply:
A for loop?

Reply:
A for loop is possible, but i think it would hurt the readability of the code. WDYT?

```
const folders = [
  [logs, logsFolderPath],
  [projectFilesToPath, projectFilesFolderPath],
  [additionalFiles, additionalFilesFolderPath],
];

const subFolders = folders.filter(item => item[0].size > 0).map(item => item[1]);
createDirectories(rootFolderPath, subFolders);
```

Reply:
Let's just leave as it is, your proposed loop could be improved to be more readable. But it's fine as it right now, maybe in the future we might require to do this.